compiler/src/org.graalvm.compiler.core.test/src/org/graalvm/compiler/core/test/GraalOptimizationTest.java

```diff
@@ -69,4 +87,9 @@ protected void checkLIR(String methodName, Predicate<LIRInstruction> predicate,
         Assert.assertEquals(expected, actualOpNum);
     }
 
+    protected void checkNode(String methodName, Class<? extends Node> expectedClass, int expected) {
```

Comment:
checkNode -> checkNodeCount?

Reply:
This name has the same style of "checkLIR". And "checkLIR" has been used in several existing testing files. So can we keep this name here?

Reply:
Well checkLIR isn't a great name either.  :) Actually looking at this again, if we remove the negative tests then the only possible argument for count is 1 so I think can can inline that argument.  I'd also inline the CLASS argument and emit it as ConditionalNode.class instead of having an alias for it.  It will use an ldc anyway so having a static final alias for it is just obfuscating I think.  In that case maybe checkConditional is a better name for it.

Reply:
Make sense! Anyway I will change this name as your suggestion! Thank you!